requirements.yml

```diff
@@ -33,3 +33,12 @@ collections:
     type: git
   - name: https://github.com/openstack/ansible-config_template
     type: git
+  - name: junipernetworks.junos
```

Comment:
IMO it would be better to match github repo reference
like
https://github.com/ansible-collections/junipernetworks.junos

Reply:
wouldn't it make more sense installing from galaxy?
I wondered why you are using git and not specifying versions

Reply:
I changed anyway but I wonder why using the git URL